templates/zerver/app/index.html

```diff
@@ -37,9 +37,20 @@
     #app-loading-middle-content h3 {
     text-align: center;
     }
+    @keyframes fadeIn {
```

Comment:
This animation is not a “fade in”. Furthermore, the `fadeIn` name conflicts with a `fadeIn` animation we declare in `static/styles/alerts.css`.

Reply:
Renamed it to `showAtEnd`